modules/nf-core/concoct/cutupfasta/main.nf

```diff
@@ -0,0 +1,40 @@
+
+process CONCOCT_CUTUPFASTA {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::concoct=1.1.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/concoct:1.1.0--py38h7be5676_2':
+        'quay.io/biocontainers/concoct:1.1.0--py38h7be5676_2' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    val(bed)
```

Comment:
The `bed` can be handled in args, right? `ext.args = {"-b ${meta.id}.bed"}`?

Reply:
File inputs should be args IMO

Reply:
Yeah good point, but using a random boolean can be restrictive IMO. but yeah nothing stops me from using it in `args` myself while specifying false for the boolean

Reply:
I was wrong! It's not a file 😱 
In that case I agree with Nicolas